Content.Server/Discord/WebhookPayload.cs

```diff
@@ -9,19 +9,19 @@ public struct WebhookPayload
     ///     The message to send in the webhook. Maximum of 2000 characters.
     /// </summary>
     [JsonPropertyName("content")]
-    public string Content { get; set; } = "";
+    public string? Content { get; set; }
 
     [JsonPropertyName("username")]
     public string? Username { get; set; }
-
+    
     [JsonPropertyName("avatar_url")]
-    public string? AvatarUrl { get; set; } = "";
+    public string? AvatarUrl { get; set; }
 
     [JsonPropertyName("embeds")]
     public List<WebhookEmbed>? Embeds { get; set; } = null;
 
     [JsonPropertyName("allowed_mentions")]
-    public WebhookMentions AllowedMentions { get; set; } = new();
+    public WebhookMentions? AllowedMentions { get; set; }
```

Comment:
If this is null then mentions will be inferred by the message's content. This means that people could force the bot to ping here or everyone by getting it to pass those strings in its message. Passing in a new empty object prevents this by default for anyone using the API.
https://discord.com/developers/docs/resources/channel#allowed-mentions-object-allowed-mentions-reference

It does mean that a new hashset is allocated every time, but WebhookMentions itself is a struct. Either way the memory use shouldn't matter.
```suggestion
    public WebhookMentions AllowedMentions { get; set; } = new();
```

Reply:
To be clear this doesn't apply for votes since those are an embed, and that is assuming that the json omits `allowed_mentions` instead of doing `allowed_mentions = null` when the value of this property is null, but either way it is safer to not rely on the options of the json serializer.

Reply:
Making the requested change here causes the vote not to send. When I've tried my program has stated that it needs to be not sent rather than just being empty- I didn't see it as an issue as admins should be trusted to not ping roles in custom votes anyways. I might be doing something wrong so if you could clarify that I'd appreciate it.